From 933b415ff58db4cf5c065db879c6c830bf3326a3 Mon Sep 17 00:00:00 2001
From: Mathieu Desnoyers <mathieu.desnoyers@efficios.com>
Date: Wed, 18 May 2011 18:42:10 -0400
Subject: [PATCH 123/248] export-vmalloc-sync-all-symbol-fix

export vmalloc_sync_all() fix

Need to export it for all architectures.

Signed-off-by: Mathieu Desnoyers <mathieu.desnoyers@polymtl.ca>
---
 mm/vmalloc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/mm/vmalloc.c b/mm/vmalloc.c
index 94dff88..9503eb2 100644
--- a/mm/vmalloc.c
+++ b/mm/vmalloc.c
@@ -2164,7 +2164,7 @@ EXPORT_SYMBOL(remap_vmalloc_range);
 void  __attribute__((weak)) vmalloc_sync_all(void)
 {
 }
-
+EXPORT_SYMBOL_GPL(vmalloc_sync_all);
 
 static int f(pte_t *pte, pgtable_t table, unsigned long addr, void *data)
 {
-- 
1.7.0.4

